sampl/madeina2
# Wire up proof of concept Slack invitations

## tl;dr

- https://madeina2slackin.herokuapp.com/

## What this pull request does

This [embeds](https://github.com/sampl/madeina2/commit/e476007d7447628d9d07e845f5956e5c1cdb7b1f) the [rauchg/slackin](https://github.com/rauchg/slackin) project into this repo and adds [a quick script](https://github.com/sampl/madeina2/blob/ac45cb2e76d3aa24becc132e0ad69d8bcd143d56/script/deploy-slackin) to deploy it to Heroku (can pull this into a grunt task if that's helpful). I have embedded the project just in an effort to keep things simple and in one place, but we can easily pull it out. We can also take this opportunity to customize the form to our liking. We can also use its iframe approach to wire it up on the main madeina2.com landing page, or more tightly integrate it. **This is just a quick proof of concept**.

## How to test

Send someone you want to invite to the Slack team to https://madeina2slackin.herokuapp.com/ and see if they can sign up.

## Caveats

Currently the app is hosted on [Heroku's free tier](https://blog.heroku.com/archives/2015/5/7/heroku-free-dynos), so it can only run for a total of 18 hours out of the day. We can look at other hosting options if we think that is too limiting.
